libs/core-ui/src/lib/Cohort/ManualCohortManagement/CohortEditorFilterSection.tsx

```diff
@@ -230,7 +251,9 @@ export class CohortEditorFilterSection extends React.PureComponent<
   };
 
   private setDefaultStateForKey(key: string): void {
-    const filter = this.getFilterValue(key);
+    const filter = this.props.isRemoveJointDatasetFlightOn
+      ? this.getFilterValue2(key)
```

Comment:
because you're going to remove the old version (ie this.getFilterValue(key)) when the flight gets deprecated, I would rename the old version to be getFilterValue2 or some function name that will be deprecated and then have the new version be whatever you want the final version name to be so you don't have to do any re-naming later